ImageNet/train/Intel_Resnet50_ec2_Xeon_8124M_intelcaffe.json

```diff
@@ -0,0 +1,31 @@
+{
+    "version": "v1.0",
+    "author": "Intel(R) Corporation",
+    "authorEmail": "daisy.deng@intel.com",
+    "framework": "Intel(R) Optimized Caffe",
+    "codeURL": "https://github.com/intel/caffe/tree/DAWN_Bench",
+    "model": "ResNet50",
+    "hardware": "64 nodes with Xeon Platinum 8124M / 144 GB / 36 Cores (Amazon EC2 [c5.18xlarge])",
+    "costPerHour": 195.84,
+    "timestamp": "2018-04-18",
+    "logFilename": "https://github.com/intel/caffe/tree/DAWN_Bench/models/intel_optimized_models/multinode/resnet_50_64_nodes_2k_batch/outputCluster-skx-64.txt.tgz",
+    "misc": {
```

Comment:
Can you add one more field to `misc` with a description of the communication method you used? Even a link to https://github.com/intel/caffe/wiki/Multinode-guide#communication from your README would be fine.